JuanDiegoGuerra/Math_Magicians
# Math Magicians: tests

### Project requirements - Tests
> Pair Programming ( @JuanDiegoGuerra & @mahammad-mostafa ) 👨🏽‍🚀
- Set up React Testing Library. ✅
  - Follow the instructions in the official [documentation](https://testing-library.com/docs/react-testing-library/intro). ✅

- Write unit tests for the files operate.js and calculate.js using Jest. ✅
- Create unit tests for all React components:
  - Use Jest snapshots to test the components. ✅
  - Use React Testing Library to simulate user interaction. ✅
- Run your full tests suite using Jest. All tests should pass. ✅
